tests/providers/google/cloud/hooks/test_pubsub.py

```diff
@@ -371,7 +371,9 @@ def test_publish(self, mock_service):
             mock.call(topic=EXPANDED_TOPIC, data=message.get("data", b''), **message.get('attributes', {}))
             for message in TEST_MESSAGES
         ]
-        publish_method.has_calls(calls)
+        mock_calls_result = publish_method.mock_calls
+        result_refined = [mock_calls_result[0],mock_calls_result[2],mock_calls_result[4]]
```

Comment:
@uranusjr For this change, I had to filter the specific calls from the mock because there are additional calls being made. Filtering allows me to do an assert on the required calls.